package.json

```diff
@@ -31,9 +31,6 @@
     "jit-grunt": "0.3.2",
     "time-grunt": "0.3.1"
   },
-  "peerDependencies": {
-    "grunt": "~0.4.0"
```

Comment:
At that point, the change would no longer be breaking, so it can be a semver PATCH upgrade